lib/spack/spack/test/install.py

```diff
@@ -275,6 +276,25 @@ class MockInstallError(spack.error.SpackError):
     pass
 
 
+def test_uninstall_by_spec_errors(mutable_database):
+    """Test exceptional cases with the uninstall command."""
+
+    # Try to uninstall a spec that has not been installed
+    rec = mutable_database.get_record('zmpi')
+    with pytest.raises(InstallError, matches="not installed"):
+        PackageBase.uninstall_by_spec(rec.spec)
+
+    # Try uninstall -- without forcing -- a spec with dependencies
+    rec = mutable_database.get_record('mpich')
+
+    rpath_args = rec.spec.package.rpath_args
+    assert rpath_args.find('-rpath') > 0
+    assert rpath_args.rfind('mpich') > 0
```

Comment:
What are these assertions checking for?

Reply:
Last-minute code coverage addition that should've been added to their own test.

Reply:
Moved the coverage tests to separate case.